packages/react-async/src/useAsync.js

```diff
@@ -154,8 +154,15 @@ const useAsync = (arg1, arg2) => {
   )
 }
 
+export class FetchError extends Error {
+  constructor(response) {
+    super(response.statusText)
```

Comment:
I'd like to have the `status` code prepended here, joined with a space. It makes it a little easier to see it originates from an HTTP request. I also think in some situations there is no `statusText`, but I'm not sure.

Reply:
Done. I think that statusText should be populated in all cases where fetch itself doesn't throw; if not, it appears to default to '', which isn't ideal but isn't terrible.